src/components/slider/index.js

```diff
@@ -250,7 +250,7 @@ export default class Slider extends PureBaseComponent {
     }
 
     if (this.minTrack) {
-      this._minTrackStyles.width = x;
+      this._minTrackStyles.width = Math.min(this.state.trackSize.width, x);
```

Comment:
I think `x` will forever be less than `width` (since the width it's the track width and the `x` is the current position of the thumb).
anyway, is this needed for some reason?

Reply:
There is an issue when you scroll from the track itself, see the attached video: 

https://user-images.githubusercontent.com/66782556/120628489-6a5ea900-c46d-11eb-8b5f-ec65079b164f.mov

Reply:
Got it, great catch :)